### ISSUE TITLE
Displaying NaN on latest update

### ISSUE THREAD

**Issue by paleoCoder** [Rytiggy/api-watchface#15]
Just pulled the latest code and now there is a new icon on the right side and a NaN displaying
![api-watchface-screenshot 20180318a](https://user-images.githubusercontent.com/272975/37574588-24643742-2adf-11e8-8d11-e8765adf4792.png)

**Comment by Rytiggy** [comment]
I haven't seen this yet, are you still experiencing this? @paleoCoder

**Comment by paleoCoder** [comment]
Yep, still have a NaN. I've reloaded the watch face a couple of times. It's probably since I have my config pointed to my nightscout site endpoint for data.

**Comment by Rytiggy** [comment]
Ahh yeah that's probably it, could you post the endpoints data ?

**Comment by paleoCoder** [comment]
the generic endpoint is: https://nightscoutsite.com/api/v1/entries.json

Here is what I get:

`[{"_id":"5ab9a15e88dff8abb5fb2daf","sgv":78,"date":1522114813000,"dateString":"2018-03-27T01:40:13.000Z","trend":4,"direction":"Flat","device":"share2","type":"sgv"},{"_id":"5ab9a03288dff8abb5fb2803","sgv":78,"date":1522114513000,"dateString":"2018-03-27T01:35:13.000Z","trend":4,"direction":"Flat","device":"share2","type":"sgv"},{"_id":"5ab99f0688dff8abb5fb2235","sgv":78,"date":1522114213000,"dateString":"2018-03-27T01:30:13.000Z","trend":4,"direction":"Flat","device":"share2","type":"sgv"},{"_id":"5ab99dda88dff8abb5fb1c7e","sgv":79,"date":1522113914000,"dateString":"2018-03-27T01:25:14.000Z","trend":4,"direction":"Flat","device":"share2","type":"sgv"},{"_id":"5ab99cad88dff8abb5fb16da","sgv":80,"date":1522113614000,"dateString":"2018-03-27T01:20:14.000Z","trend":4,"direction":"Flat","device":"share2","type":"sgv"},{"_id":"5ab99b8288dff8abb5fb1101","sgv":82,"date":1522113313000,"dateString":"2018-03-27T01:15:13.000Z","trend":4,"direction":"Flat","device":"share2","type":"sgv"},{"_id":"5ab99a5588dff8abb5fb0b49","sgv":85,"date":1522113013000,"dateString":"2018-03-27T01:10:13.000Z","trend":4,"direction":"Flat","device":"share2","type":"sgv"},{"_id":"5ab9992988dff8abb5fb05be","sgv":87,"date":1522112713000,"dateString":"2018-03-27T01:05:13.000Z","trend":4,"direction":"Flat","device":"share2","type":"sgv"},{"_id":"5ab996d188dff8abb5fafa63","sgv":92,"date":1522112113000,"dateString":"2018-03-27T00:55:13.000Z","trend":4,"direction":"Flat","device":"share2","type":"sgv"},{"_id":"5ab995a588dff8abb5faf4a4","sgv":95,"date":1522111813000,"dateString":"2018-03-27T00:50:13.000Z","trend":4,"direction":"Flat","device":"share2","type":"sgv"}]`
